tools/c7n_azure/tests/test_networksecuritygroup.py

```diff
@@ -218,7 +215,7 @@ def test_open_ports(self):
                  'match': 'any',
                  'access': 'Deny'}],
             'actions': [
-                {'type': 'open',
+                {'type': 'close',
```

Comment:
nit: what was the problem with `open`?

Reply:
Actually that action shouldn't be there at all I guess... Will revisit.